<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marketplace</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="loginEmail">
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="loginPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showRegisterButton">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="registerPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerEmail">
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerUsername">
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showLoginButton">
          <property name="text">
           <string>Back to Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="welcomeLabel">
          <property name="text">
           <string>Welcome!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logoutButton">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="productBrowsePage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Product Browse Page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
